build-system/tasks/check-exact-versions.js

```diff
@@ -35,9 +35,9 @@ function check(file) {
  * @return {!Promise}
  */
 async function checkExactVersions() {
-  const packageJsonFiles = await globby([
+  const packageJsonFiles = await fastGlob([
     '**/package.json',
-    '!**/node_modules',
+    '!**/node_modules/**',
```

Comment:
Why does this change?

Reply:
`fast-glob` requires this, while `globby` did not. We already use the convention across the rest of our globs, this was the one place where it was missing.